editoast/src/views/scenario/macro_nodes.rs

```diff
@@ -42,6 +46,22 @@ enum MacroNodeError {
     Database(#[from] editoast_models::Error),
 }
 
+impl From<ScenarioModelError> for MacroNodeError {
```

Comment:
```suggestion
impl From<models::scenario::Error> for MacroNodeError {
```

nit: we usually prefer to fully qualify (or partially at least) over creating aliases